core/src/main/scala/kafka/coordinator/group/GroupMetadataManager.scala

```diff
@@ -520,21 +520,22 @@ class GroupMetadataManager(brokerId: Int,
   def scheduleLoadGroupAndOffsets(offsetsPartition: Int, onGroupLoaded: GroupMetadata => Unit): Unit = {
     val topicPartition = new TopicPartition(Topic.GROUP_METADATA_TOPIC_NAME, offsetsPartition)
     if (addLoadingPartition(offsetsPartition)) {
+      val nowInMs = time.milliseconds()
       info(s"Scheduling loading of offsets and group metadata from $topicPartition")
-      scheduler.schedule(topicPartition.toString, () => loadGroupsAndOffsets(topicPartition, onGroupLoaded))
+      scheduler.schedule(topicPartition.toString, () => loadGroupsAndOffsets(topicPartition, onGroupLoaded, nowInMs))
     } else {
       info(s"Already loading offsets and group metadata from $topicPartition")
     }
   }
 
-  private[group] def loadGroupsAndOffsets(topicPartition: TopicPartition, onGroupLoaded: GroupMetadata => Unit): Unit = {
+  private[group] def loadGroupsAndOffsets(topicPartition: TopicPartition, onGroupLoaded: GroupMetadata => Unit, startTimeInMs: java.lang.Long = 0L): Unit = {
```

Comment:
not sure that we should default the time to 0. I think we should let the caller pass in value they want.

Reply:
Agree, done.

Reply:
(note: the reason I had left it as `0` by default was to avoid the extra dummy parameter in all the test calls, added that now)